website_seo_redirection/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Website SEO Redirection
+=======================
+
+This module extends the functionality of the website to support using custom
+URLs and allow you to improve the SEO.
+
+Usage
+=====
+
+Let's assume you created a blog post entry that is absolutely amazing and you
+want a fixed and beautiful URL for it. That entry is posted in
+``https://example.com/blog/our-news-1/post/amazing-post-23``, but you want it
+at ``https://example.com/amazing``. You need to:
+
+#. Go to *Settings > Configuration > SEO Redirections > Create*.
+#. Set ``/blog/our-news-1/post/amazing-post-23`` as *Original URL*.
+#. Set ``/amazing`` as *Redirected URL*.
+
+Now navigate to any of both URLs, and you will get to the blog post, but with
+the URL you wanted.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/8.0
+
+Known issues / Roadmap
+======================
+
+* The frontend *Promote* view should be extended to display and modify this
+  information more easily.
```

Comment:
If not, only Odoo administrators can make this changes :cry: . This is usually done by website editors without configuration permissions.